indra/llcommon/llcoros.cpp

```diff
@@ -297,7 +297,11 @@ std::string LLCoros::launch(const std::string& prefix, const callable_t& callabl
         // Out of memory on stack allocation?
         LLError::LLUserWarningMsg::showOutOfMemory();
         printActiveCoroutines();
-        LL_ERRS("LLCoros") << "Bad memory allocation in LLCoros::launch(" << prefix << ")!" << LL_ENDL;
+        LL_WARNS("LLCoros") << "Bad memory allocation in LLCoros::launch(" << prefix << ")!" << LL_ENDL;
+        if (!LLApp::isStopped() && !LLApp::isExiting())
+        {
+            LLApp::setQuitting();
+        }
```

Comment:
I updated task description, I originally picked that crash report specifically for out of disk issues, ticked has zero relation to Coro's allocation. Please keep out of memory and out of disk handling separately.